com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ArchiveBuilder.java

```diff
@@ -236,7 +238,7 @@ public void write(RandomAccessFile archiveIndex, RandomAccessFile archiveData, P
                 this.writeResourcePack(hexDigest, resourcePackDirectory.toString(), buffer, archiveEntryFlags, entry.size);
                 entries.remove(i);
             } else {
-                alignBuffer(archiveData, 4);
+                alignBuffer(archiveData, this.resourcePadding);
                 entry.resourceOffset = (int) archiveData.getFilePointer();
```

Comment:
Since the index still stores the resulting offset, it still works out of the box, and I don't think we need to update the archive format version number.